lib/cli.go

```diff
@@ -295,7 +295,7 @@ func CLI(program, version string, flags Flags) error {
 		},
 		cli.BoolFlag{
 			Name:  "no-server",
-			Usage: "do not let other servers peer with me",
+			Usage: "do not start embedded etcd server (only act as client of a cluster)",
```

Comment:
in brackets could you instead write: `(do not promote from client to peer)` ?

Reply:
also, clearify/clarify/ in commit message please.

Reply:
done